src/renderer/containers/FileDownloadDialog.tsx

```diff
@@ -0,0 +1,121 @@
+import * as React from 'react';
+import { DialogState, State } from '../state';
+import { ModalDialog } from '../components/ModalDialog';
+import { connect, DispatchProp } from 'react-redux';
+import * as actions from '../actions';
+import * as selectors from '../selectors';
+import { AnchorButton, ControlGroup, Intent } from "@blueprintjs/core";
+import { showFileOpenDialog } from "../actions";
+import { OpenDialogResult } from "../components/desktop/types";
+import { TextField } from "../components/field/TextField";
+
```

Comment:
Two lines between imports and actual code.

I wonder if we can configure `.editorconfig` to automate this.